helios-optics/src/test/kotlin/helios/optics/domain.kt

```diff
@@ -0,0 +1,25 @@
+package helios.optics
+
+import helios.meta.json
+import io.kotlintest.properties.Gen
+
+@json
+data class City(val streets: List<Street>)
+
+@json
+data class Street(val name: String)
+
+fun streetGen(): Gen<Street> = Gen.create { Street(Gen.string().generate()) }
```

Comment:
Gen.create resets the seed of Gen.string on every call. You have to extract it outside the block to keep the right entropy.

Reply:
Got this review from you twice in 2 days :D Damn.